src/EA.Iws.Web/Areas/AdminImportAssessment/Controllers/DeleteTransactionController.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace EA.Iws.Web.Areas.AdminImportAssessment.Controllers
+{
+    using System;
+    using System.Threading.Tasks;
+    using System.Web.Mvc;
+    using Infrastructure.Authorization;
+    using Prsd.Core.Mediator;
+    using Requests.ImportNotificationAssessment.Transactions;
+    using ViewModels.DeleteTransaction;
+
+    [Authorize(Roles = "internal")]
+    public class DeleteTransactionController : Controller
+    {
+        private readonly IMediator mediator;
+        private readonly AuthorizationService authorizationService;
+
+        public DeleteTransactionController(IMediator mediator, AuthorizationService authorizationService)
+        {
+            this.mediator = mediator;
+            this.authorizationService = authorizationService;
+        }
+
+        [HttpGet]
+        public async Task<ActionResult> Index(Guid id)
+        {
+            var canDeleteTransaction = await authorizationService.AuthorizeActivity(typeof(DeleteTransaction));
+
+            if (!canDeleteTransaction)
+            {
+                return RedirectToAction("Index", "AccountManagement");
+            }
```

Comment:
You can use this attribute - `[AuthorizeActivity(typeof(DeleteTransaction))]` instead of having to do the check within the action. This will return a 403 error page to users without access. This attribute can be put at class level instead of the `[Authorize(Roles = "internal")]` attribute to apply to all actions on this controller.